networkx/algorithms/triads.py

```diff
@@ -5,6 +5,8 @@
 """Functions for analyzing triads of a graph."""
 
 from networkx.utils import not_implemented_for
+from itertools import combinations, permutations
+from collections import defaultdict
 
 __all__ = ['triadic_census']
```

Comment:
Travis fails because the `__all__` var isn't updated with the new functions in the file.
should look something like this
``` python
__all__ = ['triadic_census', 'all_triplets', 'all_triads', 'triads_by_type', 'triad_type']

```

Reply:
Thanks, will fix.